test/integration/protocol_integration_test.cc

```diff
@@ -1189,6 +1191,76 @@ TEST_P(DownstreamProtocolIntegrationTest, MultipleContentLengths) {
   }
 }
 
+TEST_P(DownstreamProtocolIntegrationTest,
```

Comment:
Can you add small // comments above each new test briefly describing what is tested?